packages/component-library/components/Icon/README.mdx

```diff
@@ -15,22 +15,31 @@ import Dont from "docs-components/Dont"
+*   For clickable icons that perform actions, use an [Icon Button](/components/icon-button).
 *   Meaningful or presentational:
-    *   Use **meaningful** icons with `role=img` to convey information, such as drawing attention to an item that might have a problem using a cautionary icon: 
+    *   Use **meaningful** icons with `role=img` to convey information, such as drawing attention to an item that might have a problem using a cautionary icon:
         *   Provide a short `title` and slightly more descriptive `desc` for meaningful icons so the icon is understandable by people using a screen reader.
     *   Use **presentational** icons with `role="presentation"` for icons that do not need to convey information. They might be provided for aesthetic reasons or otherwise have meaning conveyed elsewhere.
 *   To control the icon's color, set the color property on the icon's parent element. For example, `color: $kz-color-cluny-500`.
-*   For a specific subset of icons, we include a specific icon that has the preferred color baked in, such as the destructive trash icon using Coral. Otherwise, follow the [Icons: Interaction states](https://cultureamp.design/guidelines/icons/#interaction-states) guidelines.
+*   For a specific subset of icons, we include a specific icon that has the preferred color baked in, such as the destructive trash icon using Coral or the Collective Intelligence logo. Otherwise, follow the [Icons: Interaction states](https://cultureamp.design/guidelines/icons/#interaction-states) guidelines.
+
+### Icons with Tooltips or Popovers
+
+Usually, we do not support icons receiving focus. That is, a keyboard user cannot tab to an icon. This is because icons are usually presentational or supplemented with perceivable information elsewhere.
+
+Sometimes, however, we use icons without actions that link to more information, such as a [Tooltip](/components/tooltip) or [Popover](/components/popover). These are not [Icon Buttons](/components/icon-button) because they do not have an action on click. For these icons, we ensure people can tab to the Icon component to show the Tooltip or Popover on focus as well as hover. To do this, you might add `tabindex="0"` so the icon is focusable in sequential keyboard navigation, but its order is defined by the document's source order. On focus or hover of the Icon component, you might then trigger the additional information to be displayed.
+
+### Wrapper helpers
 
 We have a helper class you can compose to utilize these styles:
 
 ```
 .myButton {
-  composes: interactiveIconWrapper from 'cultureamp-style-guide/components/Icon/Icon.module.scss';
+  composes: interactiveIconWrapper from '@kaizen/component-library/components/Icon/Icon.module.scss';
 }
 
 .myReversedButton {
-  composes: reversedInteractiveIconWrapper from 'cultureamp-style-guide/components/Icon/Icon.module.scss';
+  composes: reversedInteractiveIconWrapper from '@kaizen/component-library/components/Icon/Icon.module.scss';
```

Comment:
Just wanted to make sure the design implications of these helpers are up to date with our intentions (e.g. do we want all "interactive icons" to be 0.5 opacity by default? Could that not cause accessibility issues?). @katsambarcus or @didoesdigital would be able to confirm this.

![image](https://user-images.githubusercontent.com/6406263/73329278-2bd14100-42b1-11ea-8196-3c8bf1162ba0.png)

Reply:
@sebpearce great question. I'd love to tackle that in a follow up PR when we're ready to move to Zen.